tools/docker/tests/test_flows.sh

```diff
@@ -91,14 +91,12 @@ test_channel_selection() {
     return $check_error
 }
 
-test_ap_capability_info_reporting() {
-    #TODO: Implement
-    return 1
-}
-
-test_client_capability_info_reporting() {
```

Comment:
Was this function removed intentionally?

Reply:
Yes, the weirdly named functions were renamed to `ap_capability_query` and `client_capability_query`.